workbench-core/environments-ui/src/pages/environments/index.tsx

```diff
@@ -328,6 +328,7 @@ const EnvironmentsPage: NextPage = () => {
         )}
         {!!error && <StatusIndicator type="error">{error}</StatusIndicator>}
         <Table
+          data-testid="environmentsGrid"
```

Comment:
This is a table, should we call it `environmentTable`?

Reply:
name changed to reflect table component